luzer/CMakeLists.txt

```diff
@@ -1,14 +1,22 @@
 SetHwArchString(ARCH)
-if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if (DEFINED ENV{LIB_FUZZING_ENGINE})
```

Comment:
It would be nice to add the comment about custom naming of files in OSS-Fuzzes images. Here and below.
Feel free to ignore, though.